CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Add the visualization of labels in the visualizer (https://github.com/robotology/idyntree/pull/879)
+- Implement the `MeshcatVisualizer` python class (https://github.com/robotology/idyntree/pull/901)
```

Comment:
I think this should under a new release. As this feature is quite cool, I took the liberty of moving it myself and adding the new release already.